src/FingerprintPro.ServerSdk/Model/RawDeviceAttribute.cs

```diff
@@ -0,0 +1,114 @@
+/* 
+ * Fingerprint Pro Server API
+ *
+ * Fingerprint Pro Server API allows you to get information about visitors and about individual events in a server environment. It can be used for data exports, decision-making, and data analysis scenarios. Server API is intended for server-side usage, it's not intended to be used from the client side, whether it's a browser or a mobile device. 
+ *
+ * OpenAPI spec version: 3
+ * Contact: support@fingerprint.com
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+using System.Text;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+using FingerprintPro.ServerSdk.Json;
+
+namespace FingerprintPro.ServerSdk.Model
+{
+    /// <summary>
+    /// RawDeviceAttribute
+    /// </summary>
+    [DataContract]
+    public class RawDeviceAttribute : IEquatable<RawDeviceAttribute>
+    {
+
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RawDeviceAttribute" /> class.
+        /// </summary>
+        /// <param name="value">value.</param>
+        /// <param name="error">error.</param>
+        public RawDeviceAttribute(object value = null, RawDeviceAttributeError error = default(RawDeviceAttributeError))
+        {
+            this.Value = value;
+            this.Error = error;
+        }
+
+        /// <summary>
+        /// Gets or Sets Value
+        /// </summary>
+        [DataMember(Name = "value", EmitDefaultValue = false)]
+        [JsonPropertyName("value")]
+        public object Value { get; set; }
```

Comment:
I tried to work with the data inside Value and it feels strange, any ideas about best practice for it?

Reply:
You can check how we use it in [tests](https://github.com/fingerprintjs/fingerprint-pro-server-api-dotnet-sdk/blob/d717206b4ed179636e236dee6ff6817edbbd68bf/src/FingerprintPro.ServerSdk.Test/Api/FingerprintApiTests.cs#L312). Sadly, since it's an just object value in schema, it's probably the only way 😔